### PR TITLE
TripPlan improvement: User can upload there image from local

### PR DESCRIPTION
**Pull request by TaninDean**
workflow
[codecov](https://codecov.io/gh/ThaiRepose/thairepose)
### What is the feature?
* Allow user can upload there image from local and image will store in web folder
### What is the solution?
* Create new model to handle image
### How to test?
* Manual
### Screen shot
สกรีนช็อต 2021-11-19 080442
สกรีนช็อต 2021-11-19 080534
สกรีนช็อต 2021-11-19 080628
สกรีนช็อต 2021-11-19 080704

### CODE REVIEW

File: trip/views.py [hunk 1]
```diff
@@ -49,25 +51,33 @@ def get_queryset(self):
         return content
 
 
-class TripDetail(DetailView):
-    """Class for link html of detail of each trip."""
+def trip_detail(request, pk):
+    """Methof for link html to trip detail and add review form.
```
Comment (nabhan-au) [inline trip/views.py:55]: Miss spelling

**Comment by nabhan-au** [comment]
I think when user delete trip picture should be delete too

File: trip/views.py (hunk 1 above)
Comment (TaninDean) [reply trip/views.py]: i have fix it already.

**Comment by nabhan-au** [comment]
Can you increase coverage percentage in trip/view.py?

**Comment by TaninDean** [comment]
I have make image in local is remove when delete post already

**Comment by nabhan-au** [comment]
Capture3
I found this problem

**Review by nabhan-au — APPROVED** [review]
Nice work!!

**Pull request merged**